src/Tests/AzureClientTests.cs

```diff
@@ -404,7 +404,7 @@ public async Task TestRuntimeCapabilities(string targetId, AzureClientError? exp
             Assert.IsNotNull(azureWorkspace);
             azureWorkspace?.AddProviders("ionq", "quantinuum");
 
-            // Verify that IonQ job fails to compile (QPRGen0)
+            // Verify that IonQ job should generate warnings (QPRGen0)
```

Comment:
It's really just verifying that submission succeeds, there's no assertion for warnings.

Reply:
Updated comment.